APR-Core/QuestHandler.lua

```diff
@@ -3283,68 +3283,68 @@ APR_QH_EventFrame:SetScript("OnEvent", function(self, event, ...)
 			-- GOSSIP HARDCODED
 			if (steps and steps["Gossip"]) then
 				if (steps["Gossip"] == 27373 or steps["Gossip"] == 34398) then
-					SelectGossipOption(1)
+					C_GossipInfo.SelectOptionByIndex(1)

```

Comment:
I don't know if it's this value or -1 of the value (we keep it and we'll do a fix if ever we get a bug)